bazel/repositories.bzl

```diff
@@ -563,15 +563,11 @@ def _io_opentracing_cpp():
         actual = "@io_opentracing_cpp//:opentracing",
     )
 
-def _com_github_datadog_dd_opentracing_cpp():
-    external_http_archive("com_github_datadog_dd_opentracing_cpp")
-    external_http_archive(
-        name = "com_github_msgpack_msgpack_c",
-        build_file = "@com_github_datadog_dd_opentracing_cpp//:bazel/external/msgpack.BUILD",
-    )
```

Comment:
The `msgpack` dependency is only used by the Datadog extension being replaced and can therefore be removed too:

https://github.com/envoyproxy/envoy/blob/main/bazel/repository_locations.bzl#L732-L746